### PR TITLE
Features/django

### PR DESCRIPTION
**Pull request by mardiros**
start implementation for django.

Note that ee cannot attach event to sql on the ORM of django.

### CODE REVIEW

File: zipkin/binding/psycopg2/tracer.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import sys
+import time
+import logging
```
Comment (Themimitoof) [inline zipkin/binding/psycopg2/tracer.py:3]: Import non used.

```suggestion
```

**Comment by Themimitoof** [comment]
@mardiros don't forget to update the `setup.py` to add `django` in the `extra_requires`

**Review by Themimitoof — CHANGES_REQUESTED** [review]

File: zipkin/binding/psycopg2/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from .tracer import TraceCursor, TraceConnection
```
[the remark this answers is not in the document]
Comment (mardiros) [reply zipkin/binding/psycopg2/__init__.py]: Run Isort too

File: zipkin/binding/psycopg2/tracer.py (hunk 1 above)
Comment (mardiros) [reply zipkin/binding/psycopg2/tracer.py]: DONE

**Review by Themimitoof — APPROVED** [review]
LGTM :+1:

**Review by gdchamal — APPROVED** [review]
Just a remark about a class variable

File: zipkin/binding/psycopg2/tracer.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                    trace.record(Annotation.string('status', status))
+                    trace.record(Annotation.server_recv())
+
+        return wrapped
+    return wrapper
+
+
+class TraceCursor(_cursor):
+    """A cursor that logs queries using its connection logging facilities."""
+    count = 1
```
[the remark this answers is not in the document]
Comment (mardiros) [reply zipkin/binding/psycopg2/tracer.py:60]: Debugging purpose, to remove.

[the remark this answers is not in the document]
Comment (Themimitoof) [reply zipkin/binding/psycopg2/tracer.py:60]: Removed here: https://github.com/Gandi/python-zipkin/pull/6/files#diff-f0972ebb6e5ccf70d0677505fb969971194c1f50e4ba427580953f62c97e0c87L60

**Pull request merged**